integreat_cms/cms/models/pages/page.py

```diff
@@ -25,14 +25,24 @@ class PageQuerySet(NS_NodeQuerySet, ContentQuerySet):
     Custom queryset for pages to inherit methods from both querysets for tree nodes and content objects
     """
 
-    def cache_tree(self, archived=None):
+    def cache_tree(self, archived=None, language_slug=None):
         """
         Caches a page tree queryset in a python data structure.
 
+        Support for combinations of parameters:
+        |                  | language_slug = None | language_slug = str |
+        | ---------------- | -------------------- | ------------------- |
+        | archived = None  | yes                  | no                  |
+        | archived = True  | yes                  | no                  |
+        | archived = False | yes                  | yes                 |
+
```

Comment:
```suggestion
```
Since there is only one valid combination where `language_slug` is not `None`, I'd just mention this in the param documentation.